docs/docs/tutorials/datapane-cloud/projects.md

```diff
@@ -1,6 +1,5 @@
-# Projects
```

Comment:
Some pages, like this one, have had this kind of heading removed. 

Some haven't, e.g. workspaces:

<img width="841" alt="Screenshot 2022-10-21 at 14 28 06" src="https://user-images.githubusercontent.com/15690380/197207128-ce6b0952-6393-4b84-8c6a-84f5caeae22b.png">